### PR TITLE
Force verbatimModulesyntax

### PR DESCRIPTION
**Pull request by NullVoxPopuli**

### CODE REVIEW

File: test-app/tsconfig.json [hunk 1]
```diff
@@ -1,6 +1,13 @@
 {
   "extends": "@tsconfig/ember/tsconfig.json",
   "compilerOptions": {
+    /**
+      https://www.typescriptlang.org/tsconfig#verbatimModuleSyntax
+
+      We don't want to include types dependencies in our compiled output, so tell TypeScript
+      to enforce using `import type` instead of `import` for Types.
+     */
+    "verbatimModuleSyntax": true,
```
Comment (andreyfel) [inline test-app/tsconfig.json:10]: It is a part of the base config https://github.com/tsconfig/bases/blob/8fee7129ada13c7c779472ebb29130fa9581cf1c/bases/ember.json#L17
Why do we need to specify it here?

Comment (NullVoxPopuli) [reply test-app/tsconfig.json:10]: oh depends on which version you have installed, I forgot to check that -- its fairly recent

Comment (andreyfel) [reply test-app/tsconfig.json:10]: I've updated it to the latest in the original PR

Comment (NullVoxPopuli) [reply test-app/tsconfig.json:10]: Excellent!

**Pull request closed without merging**